### ISSUE TITLE
Enable using types downstream

### ISSUE THREAD

**Issue by adamtheturtle** [sbrunner/sphinx-prompt#394]
I notice that this project ships type stubs as it includes a `py.typed` file, and so I am hoping to use the types in a project which inherits from it (https://pypi.org/project/Sphinx-Substitution-Extensions/#optional-sphinx-prompt).

However, IIUC, I cannot access those stubs, because I have to use `__import__` to import `sphinx-prompt`.

For example:

```python
sphinx_prompt = __import__("sphinx-prompt")
PromptDirective = sphinx_prompt.PromptDirective


class SubstitutionPrompt(PromptDirective):  # type: ignore[misc, valid-type]
    ...
```

My request is that this project is changed so that I can `import sphinx_prompt` (or similar).

**Comment by sbrunner** [comment]
Effectively, it's a mistake at the project startup, should I put booth package names to keep backward compatibility?

**Comment by adamtheturtle** [comment]
@sbrunner I think that would be best.

**Comment by sbrunner** [comment]
Fixed with #397

**Comment by adamtheturtle** [comment]
Thank you for doing and releasing this.
